python/lsst/drp/tasks/gbdesAstrometricFit.py

```diff
@@ -1233,3 +1263,47 @@ def _make_outputs(self, wcsf, visitSummaryTables, exposureInfo):
             catalogs[visit] = catalog
 
         return catalogs
+
+    def _output_model_params(self, wcsf):
+        """Get the WCS model parameters and covariance and convert to a
+        dictionary that will be readable as a pandas dataframe.
```

Comment:
I'm assuming it returns a numpy ndarray by default (with column/row names?), so while I guess you can convert it to a dataframe why would you want to?

Reply:
The `ArrowNumpyDict` is a dictionary of numpy arrays. I guess I could just say that it can be converted to a table of some sort. Turning it into a dataframe is helpful for me for keeping track of what the parameters are, but maybe not the most efficient path.